hoomd/mpcd/ReverseNonequilibriumShearFlow.h

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2009-2024 The Regents of the University of Michigan.
+// Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+/*!
+ * \file ReverseNonequilibriumShearFlow.h
+ * \brief Declaration of Reverse nonequilibrium shear flow
+ */
+
+#ifndef MPCD_REVERSE_NONEQUILIBRIUM_SHEAR_FLOW_H_
+#define MPCD_REVERSE_NONEQUILIBRIUM_SHEAR_FLOW_H_
+
+#ifdef NVCC
+#error This header cannot be compiled by nvcc
+#endif
+
+#include "hoomd/ParticleGroup.h"
+#include "hoomd/Updater.h"
+#include <pybind11/pybind11.h>
+
+namespace hoomd
+    {
+namespace mpcd
+    {
+//! Reverse nonequilibrium shear flow updater
+/*!
+ * A flow is induced by swapping velocities in x direction based on particle position in
+ * y-direction.
+ */
+class PYBIND11_EXPORT ReverseNonequilibriumShearFlow : public Updater
+    {
+    public:
+    //! Constructor
+    ReverseNonequilibriumShearFlow(std::shared_ptr<SystemDefinition> sysdef,
+                                   std::shared_ptr<Trigger> trigger,
+                                   unsigned int num_swap,
+                                   Scalar slab_width,
+                                   Scalar p_target);
+
+    //! Destructor
+    virtual ~ReverseNonequilibriumShearFlow();
+
+    //! Apply velocity swaps
+    virtual void update(uint64_t timestep);
+
+    //! Set the maximum number of swapped pairs
+    void setNswap(unsigned int new_num_swap);
```

Comment:
Since this is new code, rename this method to match what we will call the Python property. Setters also don't need `new_` in the variable name because the member variable is prefixed with `m_`.
```suggestion
    //! Set the maximum number of swapped pairs
    void setNumSwap(unsigned int num_swap);
```
Make sure to change the getter & the Python binding code as well.